md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/HiveTableMigrationAPI.java

```diff
@@ -0,0 +1,369 @@
+    @ResponseBody
+    public RestWrapperOptions getdestDBList(@PathVariable("destEnv") String destEnv) {
+        LOGGER.info(destEnv + "destENV");
+        RestWrapperOptions restWrapperOptions = null;
+        try {
+            Class.forName(driverName);
+            connection = DriverManager.getConnection("jdbc:hive2://" + destEnv + "/default", "", "");
+            ResultSet rs = connection.createStatement().executeQuery("SHOW DATABASES");
+
+            List<String> databases = new ArrayList<String>();
+            while (rs.next()) {
+                String dbName = rs.getString(1);
+                databases.add(dbName.toUpperCase());
+            }
+            List<RestWrapperOptions.Option> options = new ArrayList<RestWrapperOptions.Option>();
+            for (String database : databases) {
+                RestWrapperOptions.Option option = new RestWrapperOptions.Option(database, database);
+                options.add(option);
+            }
+            restWrapperOptions = new RestWrapperOptions(options, RestWrapperOptions.OK);
+        } catch (Exception e) {
+            LOGGER.error("error occured :" + e);
+            restWrapperOptions = new RestWrapperOptions(e.getMessage(), RestWrapper.ERROR);
+        }
+        return restWrapperOptions;
+    }
+
+
+    //Fetching all the databases from hive of dest cluster
+    @RequestMapping(value = "/tables/{srcEnv}/{srcDB}", method = {RequestMethod.GET})
+    @ResponseBody
+    public RestWrapperOptions getTablesList(@PathVariable("srcEnv") String srcEnv, @PathVariable("srcDB") String srcDB) {
+        RestWrapperOptions restWrapperOptions = null;
+        LOGGER.info(srcEnv + "srcEnvi");
+        LOGGER.info(srcDB + "srcDB");
+        try {
+            Class.forName(driverName);
+            connection = DriverManager.getConnection("jdbc:hive2://" + srcEnv + "/" + srcDB.toLowerCase(), "", "");
+            ResultSet rs = connection.createStatement().executeQuery("SHOW TABLES");
+
+            List<String> databases = new ArrayList<String>();
+            while (rs.next()) {
+                String dbName = rs.getString(1);
+                databases.add(dbName.toUpperCase());
+            }
+            List<RestWrapperOptions.Option> options = new ArrayList<RestWrapperOptions.Option>();
+            for (String database : databases) {
+                RestWrapperOptions.Option option = new RestWrapperOptions.Option(database, database);
+                options.add(option);
+            }
+            restWrapperOptions = new RestWrapperOptions(options, RestWrapperOptions.OK);
+        } catch (Exception e) {
+            LOGGER.error("error occured :" + e);
+            restWrapperOptions = new RestWrapperOptions(e.getMessage(), RestWrapper.ERROR);
+        }
+        return restWrapperOptions;
+    }
+
+
+    /* protected static Connection getHiveJDBCConnection(String dbName) throws SQLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)